src/Sylius/Bundle/ApiBundle/Command/Cart/ChangeItemQuantityInCart.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command\Cart;
+
+use Sylius\Bundle\ApiBundle\Command\OrderTokenValueAwareInterface;
+
+final class ChangeItemQuantityInCart implements OrderTokenValueAwareInterface
+{
+    /** @var string|null */
+    public $orderTokenValue;
+
+    /**
+     * @var string
+     * @psalm-immutable
+     */
+    public $orderItemId;
+
+    /**
+     * @var int
+     * @psalm-immutable
+     */
+    public $newQuantity;
```

Comment:
```suggestion
    public $quantity;
```

Should be enough

Reply:
i chose `newQuantity` because this name is visible in the request body, and IMO `newQuantity` is more descriptive than just `quantity`